tests/kani/AggregateRvalue/slice_ptr.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Test codegen for a raw pointer to a struct whose last field is a slice
+
+#![feature(layout_for_ptr)]
+#![feature(ptr_metadata)]
+
+// https://github.com/model-checking/kani/issues/3615
+mod issue_3615 {
```

Comment:
Can this PR also fix #2256?

Reply:
Oh, that's likely going to fix it. I'll check it later, and add the test here if it does. Thanks!